src/plugins/next/index.ts

```diff
@@ -15,11 +15,11 @@ export const ENTRY_FILE_PATTERNS = ['next.config.{js,ts,cjs,mjs}'];
 const productionEntryFilePatternsWithoutSrc = [
   'middleware.{js,ts}',
   'app/**/route.{js,ts}',
-  'app/**/{error,layout,loading,not-found,page,template}.{js,jsx,tsx}',
+  'app/**/{error,layout,loading,not-found,page,template}.{js,jsx,ts,tsx}',
   'instrumentation.{js,ts}',
   'app/{manifest,sitemap,robots}.{js,ts}',
-  'app/**/{icon,apple-icon}-image.{js,ts,tsx}',
-  'app/**/{opengraph,twitter}-image.{js,ts,tsx}',
+  'app/**/{icon,apple-icon}-image.{js,ts,ts,tsx}',
+  'app/**/{opengraph,twitter}-image.{js,ts,ts,tsx}',
```

Comment:
This `ts` extensions is a duplicate? I read `js,ts,ts,tsx` (twice)

Reply:
If you're using `minimatch`, this pattern usually gives you all the possible extensions for a js/ts project: `?([cm])[jt]s?(x)`.

Reply:
And then some... ;) I like the explicitness and correctness here. This should only represent the extensions that Next.js (not TS) supports.

Reply:
Good catch, I think we should add jsx here.

Reply:
No need for jsx, see e.g. https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image#generate-images-using-code-js-ts-tsx